enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -622,58 +622,178 @@ will be reconciled in any namespaces, removing the need for ServiceMeshMemberRol
 are responsible for understanding and creating links to manual deployments when creating
 Gateways.
 
-The choice between these two features depends on the user's desired sharding scheme. Manual
+The choice between automated and manual deployments depends on whether a user prefers control
+over creating their own ingress gateway deployments and services, and whether they want
+multiple Gateway objects to share a single ingress gateway deployment and service. Manual
 linking, being more expressive, can establish a many-to-one Gateway-to-Gateway-Deployment
 relationship, while automated deployments strictly establish a one-to-one
 Gateway-to-Gateway-Deployment relationship. Arguably, automated deployments are more portable
 among Gateway API implementations due to the fact manually deployments require linking an
 Istio-specific service address.
 
+#### Security Policy
+
+##### Allowing All Namespaces for Gateways
+
+Unlike OpenShift ingress, neither Gateway API nor Istio restrict Gateway objects to specific namespaces. This design
+allows for different operational groups to create and manage their own [dedicated Gateway](#dedicated-gateway-topology)
+in an application namespace rather than relying on a [shared Gateway](#shared-gateway-topology) residing in another
+namespace. This departs from the current OpenShift ingress approach of all router pods being confined to the
+`openshift-ingress` namespace. Our design will allow Gateways to be created in any namespace, provided the user has
+sufficient [RBAC](#rbac) permissions.
+
+###### Limiting DNS Across Namespaces
+
+As mentioned in [New Controller to Manage DNS Records for Gateway Listeners](#new-controller-to-manage-dns-records-for-gateway-listeners),
+the Ingress Operator automatically creates DNS records for Gateways. However, for our initial implementation, this
+will be limited to the `openshift-ingress` namespace to avoid introducing complexities for DNS management while
+Gateway API upstream is still defining guidelines for DNS record management (see
+[kubernetes-sigs/gateway-api#2627](https://github.com/kubernetes-sigs/gateway-api/issues/2627)).
+
+###### Gateway Merging Across Namespaces
+
+Allowing Gateways in all namespaces can pose a potential security risk for Gateways that merge listeners
+across namespaces, as one Gateway or listener could preempt another listener in a different namespace. However,
+there currently isn't a way to merge Gateways across namespaces in Istio as the `spec.addresses`
+field is [restricted](https://github.com/istio/istio/blob/915ac2bf05dbc7a08ae61a8bc39fcd6ea1f3d11a/pilot/pkg/config/kube/gateway/context.go#L78)
+to Gateways in the same namespace. Work is underway to develop a mechanism for
+[Gateway merging](https://docs.google.com/document/d/1qj7Xog2t2fWRuzOeTsWkabUaVeOF7_2t_7appe8EXwA/edit?usp=sharing),
+and it is important to review the outcome for any potential security implications.
```

Comment:
We wont be merging listeners with Istio's current mechanism, due to it being implementation-specific and a maintenance problem:

```suggestion
```

We're looking to https://gateway-api.sigs.k8s.io/geps/gep-1713/ as the Gateway API native approach to this in the future.